app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -73,8 +73,8 @@ class MainActivity : AppCompatActivity() {
             }
         })
 
-        searchFilter.onFocusChangeListener = View.OnFocusChangeListener { view, b ->
-            if (!b)
+        searchFilter.onFocusChangeListener = View.OnFocusChangeListener { view, hasFocus ->
+            if (!hasFocus)
```

Comment:
This particular instance of ambiguous variable names is a bit more consequential, as I didn't know what this variable represented at first and had to look it up. According to https://developer.android.com/kotlin/style-guide#naming_2 this variable represents the hasFocus boolean, so I renamed "b" to to further clarify what is going on. Descriptive variable names helps reduce the need for documentation!

Reply:
Yes, spot on.